plugins/languages/markdown/glow.nix

```
@@ -0,0 +1,87 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "glow";
+  originalName = "glow.nvim";
+  defaultPackage = pkgs.vimPlugins.glow-nvim;
+
+  maintainers = [ lib.maintainers.getchoo ];
+
+  settingsOptions = {
+    glow_path = helpers.defaultNullOpts.mkStr (helpers.mkRaw "vim.fn.exepath('glow')") ''
+      Path to `glow` binary.
+
+      If null or `""`, `glow` in your $PATH with be used if available.
+
+      Consider using `glowPackage` instead.
```

```suggestion
      Using `plugins.glow.glowPackage` is the recommended way to make `glow` available on your `PATH`.
```

My suggestion can be improved a lot, but I think _something_ like this would be better?

---

That's good